### PR TITLE
Prefer option includeSubDomain to includeSubdomain

### PR DESCRIPTION
**Pull request by gedefran**
Left old option for backwards compatibility.
Updated documentation.
Added test to ensure new option is prefered.

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -116,6 +116,14 @@ describe('hsts', function () {
     assert(res.setHeader.calledWith('Strict-Transport-Security', expectedHeader));
   });
 
+  it('can include subdomains using prefered includeSubDomains option', function () {
```
Comment (EvanHahn) [inline test/index.js]: `prefered` should be `preferred`

**Comment by EvanHahn** [comment]
This is perfect—exactly what I would have done. Once you fix my typo comment, I'll merge.

If you know how to [squash commits](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html), go for it. If you don't know how, don't worry about it.

**Comment by gedefran** [comment]
All fixed up.

**Comment by EvanHahn** [comment]
Fixed in version `hsts@0.2.0` and `helmet@0.12.0`! Thanks so much.

**Comment by analytically** [comment]
Thanks guys!

**Pull request merged**